content/documentation/staging/installation-guide/index.adoc

```diff
@@ -255,6 +255,12 @@ Note that even if this fixes the hang problem, you may still have remnants of th
 
 == Additional Notes
 
+=== Versioning
+
+When you install the Kiali Operator, it will be configured to install a Kiali Server that is the same version as the operator itself. For example, if you have Kiali Operator v1.34.0 installed, that operator will install Kiali Server v1.34.0. If you want to upgrade the Kiali Server then upgrade the Kiali Operator so it can upgrade the Kiali Server.
+
+However, there are certain use-cases in which you want the Kiali Operator to install a Kiali Server whose version is different than the operator version. There are also certain use-cases when you want the Kiali Operator to install a Kiali Server whose container image is found in a different image repository or has a different image name (the default image repository and name is link:https://quay.io/repository/kiali/kiali[quay.io/kiali/kiali]). To support these use-cases, the Kiali CR provides the settings link:https://github.com/kiali/kiali-operator/blob/v1.33.1/deploy/kiali/kiali_cr.yaml#L230-L239[`spec.deployment.image_name`] and link:https://github.com/kiali/kiali-operator/blob/v1.33.1/deploy/kiali/kiali_cr.yaml#L250-L264[`spec.deployment.image_version`]. When you set one or both, the Kiali Operator will use those settings to override the defaults when it determines what Kiali Server image to install. However, note that the ability to override the image using these settings is disabled by default when you install the Kiali Operator via the Helm Chart. In order to be able to override the default Kiali Server image that will be installed by the operator, you need to set the link:https://github.com/kiali/helm-charts/blob/v1.33.1/kiali-operator/values.yaml#L51-L55["allow ad-hoc kiali image"] helm value to `true`. If you already have an operator installed and you want to enable this feature, you can do so by following the instructions found at link:../faq/#operator-configuration[FAQ: How to configure some operator features at runtime].
```

Comment:
A little wordy and maybe indicate why we disable this by default:   "However, for security reasons these settings are disabled by default,  when you install the Kiali Operator via the Helm Chart.  To allow the operator to install a non-default Kiali Server image  you must set..."